properties/base/instance.py

```diff
@@ -9,7 +9,7 @@
 
 from six import PY2
 
-from .base import HasProperties, equal
+from ..base import GENERIC_ERRORS, HasProperties, equal
```

Comment:
Silly question, but should this be `from . import GENERIC_ERRORS, HasProperties, equal`? I mean, if it's in `..base` then presumably it keys the import through this directory's `__init__.py`?
This line seems odd.

Reply:
Great point... not sure how I landed on this line...